NiBAx/plugins/loadsave/dataio.py

```diff
@@ -36,9 +36,21 @@ def ReadMUSEDictionary(self):
         # e.g. ID to name; MUSE_Volume_48 -> Hippocampus right
         MUSEDictIDtoNAME = dict(zip(MUSEDict['ROI_COL'], MUSEDict['ROI_NAME']))
 
+        MUSEDictDataFrame = MUSEDict
+
         logger.info('MUSE dictionary read from file: %s', MUSEDictfile)
 
-        return MUSEDictNAMEtoID, MUSEDictIDtoNAME
+        return MUSEDictNAMEtoID, MUSEDictIDtoNAME, MUSEDictDataFrame
+    
+    def ReadDerivedMUSEMap(self):
+        # Load MUSE dictionary file
+        DerivedMapfile = os.path.join(os.path.dirname(__file__), 'MUSE_DerivedROIs_Mappings.csv')
+        DerivedMUSEMap = pd.read_csv(DerivedMapfile,
+        header=None, engine='python').reset_index().rename(columns={'level_0':'ROI_INDEX','level_1':'ROI_NAME'}).set_index('ROI_INDEX').drop(columns=['ROI_NAME'])
```

Comment:
The proposed implementation did not work on Windows 10 Python 3.8.8. Here is an alternative that will.

```suggestion
        DerivedMUSEMap = pd.read_csv(DerivedMapfile,
        header=0, engine='python').reset_index().rename(columns={'level_0':'ROI_INDEX','level_1':'ROI_NAME'}).set_index('ROI_INDEX').drop(columns=['ROI_NAME'])
```

Reply:
Correction. While the file will be read without error, it takes the entries of the first line as column names, thus resulting in `['702', 'ICV']`.

Reply:
New suggestion. Loads data frame, but then also leads to crash because it cannot find key `301`.

```suggestion
        DerivedMUSEMap  = pd.read_csv(DerivedMapfile, sep='^', header=None)
        DerivedMUSEMap2 = DerivedMUSEMap[0].str.split(',', expand=True)
        del DerivedMUSEMap[0]
        DerivedMUSEMap  = pd.concat([DerivedMUSEMap, DerivedMUSEMap2], axis=1)
        del DerivedMUSEMap2
        DerivedMUSEMap  = DerivedMUSEMap.rename(columns={0:'ROI_INDEX', 1:'ROI_NAME'}).set_index('ROI_INDEX').drop(columns=['ROI_NAME'])
```

Reply:
The proposed PR worked with `pandas==1.3.4`, but did not with `pandas==1.4.1`.